selfhosted-swagger.json

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
and i guess could also rename the file to apiserver-swagger.json

Reply:
done